src/home/index.js

```diff
@@ -5,14 +5,16 @@ import ItemPreview from "./ItemPreview";
 
 function Home() {
   return (
+      <div className="container">
       <div className="row w-100">
-        <div className="col-lg-4 d-none d-lg-block">
+        <div className="col-lg-4 col-xl-4 col-xxl-4 d-none d-lg-block">
```

Comment:
the size based classes are cascading, so if you have the same number of columns for several sequential sizes, you can simplify. In this case, this is the same as "col-lg-4 d-none d-lg-block"